explanation/about-explanation.md

```diff
@@ -1,54 +1,48 @@
 # The explanation article
 
-## When do I need an explanation article?
+Explanation or Discussion articles explain background information and provide context-specific knowledge on a particular topic.
```

Comment:
Probably lowercase "discussion" here. 

There are probably several places where these appear as proper nouns because we set them up like this before adopting a style guide.

Reply:
If we're treating template titles as proper nouns elsewhere, shouldn't a discussion article also be treated as a proper noun?